trimChamber.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python
+#!/bin/env python2.7
```

Comment:
@cbravo135 I don't understand. This routine (trimChamber.py) is what is currently used in p5 to perform the trimming procedure is it not?